.github/renovate.json5

```diff
@@ -11,4 +11,7 @@
     },
   ],
   ignorePaths: ["plugins/destination/csv/**"],
+  gomod: {
+    fileMatch: ["go\\.mod\\.tpl$"],
```

Comment:
I hope this works, as renovate also does some tidying after the update.
If it doesn't I'll need to find a way to exclude the template file from the tidying part

Reply:
It works https://github.com/cloudquery/cloudquery/pull/7121/files#diff-24b7ec064fbad86a0563aef540f40e702d9797a62f53a834d126e797002f4665R6